src/components/ActivityButton.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button } from '@mui/material';
+
+type ActivityButtonProps = {
+    color?: "inherit" | "primary" | "secondary";
+    onClick: () => void;
+    children?: React.ReactNode
+}
+
+const ActivityButton = ({color, onClick, children}: ActivityButtonProps): JSX.Element => {
```

Comment:
Minor comment: as discussed, maybe this is not needed to be extracted?
And you could just use a normal button instead of this, but that's up to you!

Reply:
Made an `AddActivityButton` instead.
(Or should I not make a new button component ? 🤔 )

Reply:
I think it's ok to have it as a separate button for now.